### ISSUE TITLE
addr:neighbourhood and addr:suburb

### ISSUE THREAD

**Issue by contrapunctus-1** [Zverik/every_door#663]
In addition to [addr:block](https://github.com/Zverik/every_door/issues/455#issuecomment-1763369436), it is very common for addresses in India to use components which (based on community feedback) are best tagged using `addr:neighbourhood` and `addr:suburb`.

For example -
A-86, Gali No.1, Rajeev Nagar, Bhalaswa, Delhi
addr:block=A
addr:housenumber=86
addr:street=Gali Number 1
addr:neighbourhood=Rajeev Nagar
addr:suburb=Bhalaswa
addr:city=Delhi

Using the raw tags editor for these fields requires users to know the OSM tags to use, which defeats the purpose of using EveryDoor.

Please add support for `addr:neighbourhood` and `addr:suburb`. Thank you.

**Comment by pandadeepimpact** [comment]
I agree with the difficulty of using raw tags, but as countries have varying formats for addresses, I think it would be better if some fields can be configured to appear only on certain countries. Or an alternative is to put these country-specific address tags in a dropdown menu. That would avoid cluttering the screen yet keep the speed advantage of using Every Door.
